spec/integration/put_content/content_with_embedded_content_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe "PUT /v2/content when embedded content is provided" do
+  include_context "PutContent call"
+
+  context "with embedded content" do
+    let(:contact) { create(:edition, state: "published", content_store: "live", document_type: "contact") }
+    let(:document) { create(:document) }
+    let!(:link) { document.content_id }
```

Comment:
We should have a test where we already have some existing links in the payload to check they're being merged correctly.